### PR TITLE
docs: Update resources.md

### PR DESCRIPTION
**Pull request by yzgyyang**
## Summary of Changes

- Fix the path to the simulator
- Title & Punctuation improvement

## Additional context

Discord username (if different from GitHub): yzgyyang#3140

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: docs/resources.md [hunk 1]
```diff
@@ -16,15 +16,16 @@
...
+
+You can create a symlink (or "directory junction" in Windows terms) between your local repo and the game's community folder. This lets you develop from your local repository without having to copy files over when you're ready to test. Developers working on hot-reloadable files can get changes reflected in real-time, while still retaining a proper development environment.
 
 First, be sure to remove any existing install of the mod from the game.
 
 Then in `cmd`, run the command below. Be sure to replace each path with the proper respective file path.
 
 ```cmd
-mklink /J "C:\users\<user>\AppData\Microsoft Flight Simulator\Packages\Community\flybywire-aircraft-a320-neo" "C:\path\to\cloned\repo\flybywire-aircraft-a320-neo"
+mklink /J "C:\Users\<user>\AppData\Roaming\Microsoft Flight Simulator\Packages\Community\flybywire-aircraft-a320-neo" "C:\path\to\cloned\repo\flybywire-aircraft-a320-neo"
```
Comment (scoopscode) [inline docs/resources.md:28]: Path is `\AppData\Local\` not `\AppData\Roaming\`

Comment (yzgyyang) [reply docs/resources.md:28]: It is under `Roaming` for my installation - maybe because I have installed it using Steam?

Comment (scoopscode) [reply docs/resources.md:28]: Yeah thats why, maybe you could specify for each edition?

Comment (sidnov) [reply docs/resources.md:28]: ```\AppData\Roaming\``` is the path for Steam. Can someone confirm for Store edition

Comment (Valastiri) [reply docs/resources.md:28]: I have local listed on our docs site -> [here](https://docs.flybywiresim.com/start/installation/#microsoft-store-andor-game-pass-edition) for Store edition. This was also after an update to both our repo README.md and the docs site. Should be valid.

**Comment by davidwalschots** [comment]
I'm going to close this due to being inactive in addressing the review comment.

**Pull request closed without merging**